js/gated-content/src/router/index.js

```diff
@@ -90,12 +104,24 @@ const routes = [
     meta: { requiresAuth: true },
   },
   {
-    path: '/categories',
-    redirect: '/categories/video',
+    path: '/blog-post',
+    redirect: { name: 'CategoryListing', query: { type: 'vy_blog_post' } },
```

Comment:
Probably should be redirected to BlogsListing?

Reply:
yep, nice catch!

Reply:
Fixed